src/index.js

```diff
@@ -62,6 +62,26 @@ module.exports = (web3, options = {}) => {
     version.split('.').map((part) => parseInt(part))
 
   const getApplicationInfo = (contentURI) => {
+    let errorMsg
+    let reportType
+    const [provider, location] = contentURI.split(':')
+
+    if (!provider || !location) {
+      errorMsg = `contentURI: ${contentURI} is invalid.`
+      reportType = 'warning'

```

Comment:
I think we can remove `reportType` from here and always report a warning in the CLI.